src/imitation/util/logger.py

```diff
@@ -3,23 +3,70 @@
 import contextlib
 import datetime
 import os
+import sys
 import tempfile
 from typing import Any, Dict, Generator, Optional, Sequence, Tuple, Union
 
-import stable_baselines3.common.logger as sb_logger
+from stable_baselines3.common.logger import (
+    CSVOutputFormat,
+    HumanOutputFormat,
+    JSONOutputFormat,
+    KVWriter,
+    Logger,
+    TensorBoardOutputFormat,
+)
 
 from imitation.data import types
 
 
+def make_output_format(
+    _format: str,
+    log_dir: str,
+    log_suffix: str = "",
+    max_length: int = 40,
+) -> KVWriter:
+    """Returns a logger for the requested format.
+
+    Args:
+        _format: the requested format to log to
+            ('stdout', 'log', 'json' or 'csv' or 'tensorboard').
+        log_dir: the logging directory.
+        log_suffix: the suffix for the log file.
+        max_length: the maximum length beyond which the keys get truncated.
+
+    Returns:
+        the logger.
+
+    Raises:
+        ValueError: if `_format` is unknown.
+    """
+    os.makedirs(log_dir, exist_ok=True)
+    if _format == "stdout":
+        return HumanOutputFormat(sys.stdout, max_length=max_length)
+    elif _format == "log":
+        return HumanOutputFormat(
+            os.path.join(log_dir, f"log{log_suffix}.txt"),
+            max_length=max_length,
+        )
+    elif _format == "json":
```

Comment:
These and following lines are identical to that in `sb_logger.make_output_format` when using `HumanOutputFormat`. We could make code more DRY by having e.g.

```
else:
    return sb_logger.make_output_format(_format, log_dir, log_suffix)
```

WDYT?

Reply:
Great suggestion! Should've thought of that.